.github/workflows/ci_lin.yml

```diff
@@ -67,7 +67,7 @@ jobs:
           shell: bash
           command: |
             ARCHIVE_PATH=$(cat $GITHUB_WORKSPACE/../../../../GH_CI_LIN_ARCHIVE_FOLDER.txt)
-            find $ARCHIVE_PATH -maxdepth 1 -type d -name "*_*0000" -print | sort -r | head -1 |xargs -i cp -r "{}/build/linux_prod/" ./
+            rsync -av "${ARCHIVE_PATH}/latest/build/*" ./
```

Comment:
I have a concern about this change.
If we use "latest" instead of a date, how can we check the version of the compiler/dpct/dependencies when doing failure analysis?